### ISSUE TITLE
Fractional Regression: `frm` package

### ISSUE THREAD

**Issue by briatte** [m-clark/m-clark.github.io#2]
Hello,

Re: your fractional regression tutorial,

https://m-clark.github.io/posts/2019-08-20-fractional-regression/

Two very different questions:

1. Are you familiar with the `frm` packages, which exist both for R and Stata?

https://cran.r-project.org/package=frm
https://cran.r-project.org/package=frmhet
https://cran.r-project.org/package=frmpd

The author, Joaquim J.S. Ramalho, has a few papers associated with the packages:

https://doi.org/10.1111/j.1467-6419.2009.00602.x
https://doi.org/10.1080/07474938.2014.976531
https://doi.org/10.1515/jem-2015-0019

2. If I want to find your frac. reg. on your website via an index, where should I look? On this page perhaps (although I don't see it listed there yet)?

https://m-clark.github.io/documents.html

P.S. You work on documenting R models is awesome.

**Comment by m-clark** [comment]
I just posted that and am going back through the content to (hopefully) make it bit more accessible, and am working with some feedback such as this, so thanks.  I am aware of `frm`, and had a footnote about it, but I hesitated to provide an example due to the non-formula interface, lack of basic modeling methods, etc.  As such, I figured would it not be as easy for others as the approach with glm/sandwich for the standard setting (and I kind of wanted to advertise the non-magical nature of it).  Perhaps it might be better to go ahead and give the demo as its own section, or if nothing else, I will add those papers to the references. 

The post will be listed in the documents section when I can get around to that (soon).  I'm also now using the 'categories' listings so that content can potentially be found/filtered on the front page.  Feel free to check that out.

And thanks for the kind words! Always glad to hear the docs are useful!
